lib/library.rb

```diff
@@ -0,0 +1,32 @@
+class Library
+
+  attr_reader :name, :books, :authors
+
+  def initialize(name)
+    @name = name
+    @books = []
+    @authors = []
+  end
+
+  def add_author(author)
+    @authors << author
```

Comment:
now that our author is in the library, how can we add their books to our library?